### PR TITLE
Add docs on cross-cluster listing endpoints

### PR DESCRIPTION
**Pull request by cipherboy**
`Signed-off-by: Alexander Scheel <alex.scheel@hashicorp.com>`

### CODE REVIEW

File: website/content/api-docs/secret/pki.mdx [hunk 1]
```diff
@@ -1007,6 +1009,79 @@ $ curl \
...
+      "3d:80:91:c3:c2:34:3b:81:69:3d:92:a3:80:69:db:53:04:26:ab:b4"
+    ]
+  }
+}
+```
+
+### List Cross-Cluster Revocations
+
+This endpoint returns a list of serial numbers that have been revoked on any
+cluster, along with a cluster that has a copy of that revoked certificate.
```
Comment (stevendpclark) [inline website/content/api-docs/secret/pki.mdx:1051]: nit:
```suggestion
cluster, along with the clusters that have a copy of that revoked certificate.
```

**Review by stevendpclark — APPROVED** [review]

**Review by yhyakuna — APPROVED** [review]

File: website/content/api-docs/secret/pki.mdx (hunk 1 above)
Comment (cipherboy) [reply website/content/api-docs/secret/pki.mdx:1051]: I thought we decided that this API would only return a single cluster, not all clusters.

Comment (stevendpclark) [reply website/content/api-docs/secret/pki.mdx:1051]: That was the cross-cluster queue list, the revocation one returns [them all](https://github.com/hashicorp/vault/blob/e7636b47c6fd5261ab2f08d07971c23a0a6a937e/builtin/logical/pki/path_revoke.go#L622-L658).

Comment (cipherboy) [reply website/content/api-docs/secret/pki.mdx:1051]: Ah, I see.

**Pull request merged**